.gitignore

```diff
@@ -1 +1,2 @@
 test.py
+LSP/
```

Comment:
Did you clone the LSP repo in the LSP-rust-analyzer repo? You can also just clone it into ST's Packages/ folder and LSP-pyright will pick it up.

Reply:
Oh I had no idea. Wow ty!